homework/Repository.js

```diff
@@ -8,25 +8,34 @@ class Repository {
     this.repository = repository;
   }
 
-  /**
-   * Render the repository info to the DOM.
-   * @param {HTMLElement} container The container element in which to render the repository.
-   */
   render(container) {
-    // TODO: replace the next line with your code.
-    Util.createAndAppend('pre', container, JSON.stringify(this.repository, null, 2));
+    const theads = ['Repository:', 'Description:', 'Forks:', 'Updated:'];
+    for (let i = 0; i < theads.length; i++) {
+      const tr = Util.createAndAppend('tr', container);
+      Util.createAndAppend('th', tr, { text: theads[i] });
+      const tds = ['name', 'description', 'forks', 'updated_at'];
+      if (tds[i] === 'name') {
+        const linked = Util.createAndAppend('td', tr);
+        Util.createAndAppend('a', linked, {
+          id: 'linked',
+          text: this.repository[tds[i]],
+          href: this.repository.html_url,
+          target: '_blank',
+          role: `Repository name is ${this.repository.name}`,
```

Comment:
ARIA Role represents the type of the element, see the list of roles supported to choose the proper one to your case
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques#Roles

Reply:
I changed it as you mentioned. Thanks for the feedback.